libs/mrwebrtc/src/interop/global_factory.cpp

```diff
@@ -251,6 +252,8 @@ mrsResult GlobalFactory::InitializeImplNoLock() {
     factoryConfig->audioCapturingEnabled = true;
     factoryConfig->audioRenderingEnabled = true;
     factoryConfig->enableAudioBufferEvents = false;
+    factoryConfig->customAudioMixer =
+        reinterpret_cast<uint64_t>(custom_audio_mixer_.get());
```

Comment:
Probably won't build in 32-bit archs

Reply:
It turns out it is correct but produces a warning. Fixed.